webapp/src/components/form/FormTextArea.js

```diff
@@ -12,6 +12,27 @@ class FormTextArea extends React.Component {
         this.nameInput.focus()
       }
     }
+
+    getWordCount = () => {
+      if (this.props.value) {
+        const length= this.props.value.split(" ").filter(value => (value!== "")).length;
+        if(length<10){
```

Comment:
I'm guessing these console logs were for debugging, can you remove them please?

Reply:
Done.